Model/AutoRoute.php

```diff
@@ -50,7 +50,7 @@ public function setType($type)
         $this->setDefault('type', $type);
     }
 
-    public function setRedirectTarget(AutoRouteInterface $redirectRoute)
+    public function setRedirectTarget($redirectRoute)
```

Comment:
No, you can just [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) on master:

````
$ git fetch origin
$ git rebase origin/master
````